src/components/ammo-constraint.js

```diff
@@ -0,0 +1,141 @@
+/* global Ammo */
+
+module.exports = AFRAME.registerComponent("ammo-constraint", {
+  multiple: true,
+
+  schema: {
+    // Type of constraint.
+    type: { default: "lock", oneOf: ["lock", "fixed", "spring", "slider", "hinge", "coneTwist", "pointToPoint"] },
+
+    // Target (other) body for the constraint.
+    target: { type: "selector" },
+
+    // TODO: Wake up bodies when connected.
+    wakeUpBodies: { default: true },
```

Comment:
Is the effect here just that the constraint wakes the bodies when it's created? Or does it also have some effect like keeping them awake over time? Trying to decide the right way to name this.

Reply:
Actually I don't think this needs to be here- it was just copied over from the original constraint. I have a `TODO` because I thought I would need to implement it but it doesn't look to be the case.

Reply:
Ah ok!